plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -36,6 +45,86 @@ const containerRuntimes = {
 };
 
 class GeneralServiceFormSection extends Component {
+  constructor() {
+    super(...arguments);
+
+    this.state = {convertToPodModalOpen: false};
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  handleConvertToPod() {
+    this.props.onConvertToPod();
+    this.handleCloseConvertToPodModal();
+  }
+
+  handleCloseConvertToPodModal() {
+    this.setState({convertToPodModalOpen: false});
+  }
+
+  handleOpenConvertToPodModal() {
+    this.setState({convertToPodModalOpen: true});
+  }
+
+  getContainerSection() {
+    let {data = {}, errors} = this.props;
+
+    if (!(this.props.service instanceof Pod)) {
+      return (
+        <ContainerServiceFormSection
+          data={data}
+          errors={errors.container}
+          onAddItem={this.props.onAddItem}
+          onRemoveItem={this.props.onRemoveItem} />
+      );
+    }
+
+    let {containers = []} = data;
+    return containers.map((item, index) => {
+      return (
+        <FormGroupContainer key={index}
+          onRemove={this.props.onRemoveItem.bind(this, {value: index, path: 'containers'})}>
+          {item.name || `container ${index + 1}`}
```

Comment:
Let's remove the space between `container` and `index`, so we don't get an error right from adding a new container```{item.name || `container${index + 1}`}```